lib/util.js

```diff
@@ -504,49 +505,60 @@ const isArray = exports.isArray = Array.isArray;
 function isBoolean(arg) {
   return typeof arg === 'boolean';
 }
-exports.isBoolean = isBoolean;
+exports.isBoolean = deprecate(isBoolean
+  , 'util.is* functions are deprecated, please use a user-land alternative.');
```

Comment:
I think its better to be explicit here. If someone is using additional is*, it'll bubble up somewhere else.